packages/components/src/components/NFT/NFT.tsx

```diff
@@ -15,24 +22,66 @@ export interface NFTProps {
  * TODO: Add a component that fetchs the NFT data from the blockchain and uses this component to
  * display it
  */
-export const NFT = ({
-  tokenId,
-  name,
-  imageUrl,
-  assetContractName,
-  assetContractSymbol,
-}: NFTProps) => {
-  const displayName = name || tokenId;
+export const NFT = ({ contractAddress, tokenId }: NFTProps) => {
```

Comment:
do you think we could take the approach i mentioned here for the NFT component:https://github.com/Developer-DAO/web3-ui/pull/46#issue-1068068696
>Add a wrapper around NFT (and make the current implementation private) that uses a tokenId and contract address to fetch metadata about it

We should keep the existing component around and use it for the `NFTGallery` component so we don't need to refetch the NFT data that already comes from the previous API

Reply:
@etr2460 missed that. That sounds fair